aicsimageio/readers/ome_zarr_reader.py

```diff
@@ -135,6 +157,26 @@ def _get_coords(
 
         return coords
 
+    @staticmethod
+    def _get_pixel_size(
+        reader: ZarrReader, dims: List[str], series_index: int, resolution_index: int
+    ) -> Tuple[Optional[float], Optional[float], Optional[float]]:
+
+        coord_transform = reader.root_attrs["multiscales"][series_index]["datasets"][
+            resolution_index
+        ]["coordinateTransformations"]
+
+        spatial_coeffs = {}
```

Comment:
This is totally just a preference so nbd either way.

Since you are looking specifically for three variables it might be more readable IMO to avoid doing this iteratively and instead maybe like so:
```
spatial_dim_scales = reader.root_attrs["multiscales"][series_index]["datasets"][
            resolution_index
        ]["coordinateTransformations"][0]["scale"]
x_physical_size = spatial_dim_scales[dims.index(DimensionNames.SpatialX)] if DimensionNames.SpatialX in  spatial_dim_scales else None
y_physical_size = spatial_dim_scales[dims.index(DimensionNames.SpatialY)] if DimensionNames.SpatialY in  spatial_dim_scales else None
z_physical_size = spatial_dim_scales[dims.index(DimensionNames.SpatialZ)] if DimensionNames.SpatialZ in  spatial_dim_scales else None
return (z_physical_size, y_physical_size, x_physical_size)
```